shdb/service.go

```diff
@@ -52,6 +52,7 @@ type Service interface {
 	// Identity
 	TouchIdentity(string, *sql.Tx) (int64, error)
 	GetIdentity(string, bool, *sql.Tx) (*models.IdentityDataOutput, error)
+	GetIdentityByUser(key string, value string, missingFatal bool, tx *sql.Tx) (identityData *models.IdentityDataOutput, err error)
```

Comment:
You don't need to put parameters names here in a declaration - let's be consistent with the remaining methods, so the signature would be:
```
GetIdentityByUser(string, string, bool, *sql.Tx) (*models.IdentityDataOutput, error)
```